README.adoc

```diff
@@ -17,6 +17,8 @@
 :source-highlighter: prettify
 :figure-caption!:
 = Injecting dependencies into microservices
+[.hidden]
```

Comment:
Can you add a space here to make it clearer. Thanks

Reply:
Sure I will